hw/ip/rom_ctrl/data/rom_ctrl.hjson

```diff
@@ -143,6 +143,14 @@
         integrity scheme).
       '''
     },
+    { name: "BUS.LOCAL_ESC",
```

Comment:
@msfschaffner, @tjaychen: Thoughts about the naming here? My logic was that this is protecting bus data (from unexpected responses or rdata) by locally escalating an "alert" signal. But the name looks pretty weird!

Reply:
i think that's fine. Especially since you have quite a lot of description below.